frontend/test/metabase/scenarios/query_builder.cy.spec.js

```diff
@@ -0,0 +1,96 @@
+import { signInAsAdmin } from "__support__/cypress";
+
+describe("query builder", () => {
+  beforeEach(signInAsAdmin);
+
+  describe("browse data", () => {
+    it("should load orders table and summarize", () => {
+      cy.visit("/");
+      cy.contains("Browse Data").click();
+      cy.contains("Sample Dataset").click();
+      cy.contains("Orders").click();
+      cy.contains("37.65");
+    });
+  });
+  describe("ask a (simple) question", () => {
+    it("should load orders table", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Simple question").click();
+      maybeClickSampleDataset();
+      cy.contains("Orders").click();
+      cy.contains("37.65");
+    });
+
+    it("should load orders table", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Custom question").click();
+      maybeClickSampleDataset();
+      cy.contains("Orders").click();
+      cy.contains("Visualize").click();
+      cy.contains("37.65");
+    });
+
+    it("should summarize and break out and show a map", () => {
+      cy.visit("/");
+      cy.contains("Ask a question").click();
+      cy.contains("Custom question").click();
+      maybeClickSampleDataset();
+      cy.contains("Orders").click();
+      cy.contains("Pick the metric you want to see").click();
+      cy.contains("Count of rows").click();
+      cy.contains("Pick a column to group by").click();
+      cy.contains(/^User$/).click();
+      cy.contains("State").click();
+      cy.contains("Visualize").click();
+      cy.contains("1,342 +");
+    });
+  });
+
+  describe("view mode", () => {
+    describe("summarize sidebar", () => {
+      it("should summarize by category and show a bar chart", () => {
+        cy.server();
+        cy.route("POST", "/api/dataset").as("dataset");
+        loadOrdersTable();
+        cy.wait("@dataset");
```

Comment:
Why did you need to wait on this API call for summarize but not filter?

Reply:
Honestly... I don't fully understand why it's necessary.

Reply:
I think I saw the issue when I throttled my network right before clicking "Orders". Something weird happens in the summarize sidebar. Maybe "Category" gets removed and readded quickly or something?

If we do need this wait, would `cy.contains("37.65")`  (or something else in the orders table) have the same effect as `cy.wait("@dataset")`? That'd be one line rather than three, but it's less clearly why we're doing it.